FireboltDotNetSdk.Tests/Integration/ConnectionTest.cs

```diff
@@ -0,0 +1,47 @@
+using FireboltDotNetSdk.Client;
+
+
+namespace FireboltDotNetSdk.Tests
+{
+
+    [TestFixture]
+    internal class ConnectionTest : IntegrationTest
+    {
+        [Test]
+        public void ConnectNoEngineTest()
+        {
+            var connString = $"clientid={ClientId};clientsecret={ClientSecret};account={Account};env={Env}";
+            FireboltConnection Connection = new FireboltConnection(connString);
+            Connection.Open();
+            var cursor = Connection.CreateCursor();
+            cursor.Execute("SELECT 1");
+            Assert.NotNull(cursor.Response);
+            NewMeta newMeta = ResponseUtilities.getFirstRow(cursor.Response!);
+            Assert.That(newMeta.Data[0], Is.EqualTo(1));
+        }
+        [Test]
+        public void ConnectNoEngineWithDatabaseTest()
+        {
+            var connString = $"database={Database};clientid={ClientId};clientsecret={ClientSecret};account={Account};env={Env}";
+            FireboltConnection Connection = new FireboltConnection(connString);
+            Connection.Open();
+            var cursor = Connection.CreateCursor();
+            cursor.Execute("SELECT 1");
```

Comment:
Same as in some other connector, I think it's worth testing relevant sql for each test case. e.g. `SELECT TOP 1 * FROM information_schema.tables` would only work if we have a database context. etc.